Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/Lib/Helpers.cs

```diff
@@ -169,5 +140,115 @@ internal static void SetTestFileLastModified(string path, DateTime date)
             client.Disconnect();
         }
     }
+
+    internal static string GetServerFingerprintAsMD5HexString()
+    {
+        var fingerprint = "";
+        using (var client = new SftpClient(_dockerAddress, 2222, _dockerUsername, _dockerPassword))
+        {
+            client.HostKeyReceived += delegate (object sender, HostKeyEventArgs e)
+            {
+                fingerprint = BitConverter.ToString(e.FingerPrint).Replace("-", ":");
+                e.CanTrust = true;
+            };
+            client.Connect();
+            client.Disconnect();
+        }
+
+        return fingerprint;
+    }
+
+    internal static string GetServerFingerprintAsSHA256Base64String()
+    {
+        var fingerprint = "";
+        using (var client = new SftpClient(_dockerAddress, 2222, _dockerUsername, _dockerPassword))
+        {
+            client.HostKeyReceived += delegate (object sender, HostKeyEventArgs e)
+            {
+                using (SHA256 mySHA256 = SHA256.Create())
+                {
+                    fingerprint = Convert.ToBase64String(mySHA256.ComputeHash(e.HostKey));
+                }
+                e.CanTrust = true;
+            };
+            client.Connect();
+            client.Disconnect();
+        }
+
+        return fingerprint;
+    }
+
+    internal static string GetServerFingerprintAsSHA256HexString()
+    {
+        var fingerprint = "";
+        using (var client = new SftpClient(_dockerAddress, 2222, _dockerUsername, _dockerPassword))
+        {
+            client.HostKeyReceived += delegate (object sender, HostKeyEventArgs e)
+            {
+                using (SHA256 mySHA256 = SHA256.Create())
+                {
+                    fingerprint = ToHex(mySHA256.ComputeHash(e.HostKey));
+                }
+                e.CanTrust = true;
+            };
+            client.Connect();
+            client.Disconnect();
+        }
+
+        return fingerprint;
+    }
+
+    internal static string GetServerFingerprintAsSHA1String()
+    {
+        var fingerprint = "";
+        using (var client = new SftpClient(_dockerAddress, 2222, _dockerUsername, _dockerPassword))
+        {
+            client.HostKeyReceived += delegate (object sender, HostKeyEventArgs e)
+            {
+                using (var sha1 = SHA1.Create())
+                {
+                    var hash = sha1.ComputeHash(e.HostKey);
+                    fingerprint = string.Concat(hash.Select(b => b.ToString("x2")));
+                }
+                e.CanTrust = true;
+            };
+            client.Connect();
+            client.Disconnect();
+        }
+
+        return fingerprint;
+    }
+
+    internal static string HexStringToB64String(string input)
+    {
+        return Convert.ToBase64String(ConvertHexStringToHex(input));
+    }
+
+    internal static byte[] ConvertHexStringToHex(string hex)
```

Comment:
Should this be "ConvertHexStringToByteArray"?

Reply:
That method was not used in anything so I removed that altogether.